packages/blockchain-wallet-v4-frontend/src/data/preferences/sagas.ts

```diff
@@ -20,7 +20,16 @@ export default () => {
     }
   }
 
+  const setLinkHandling = function () {
+    // Register BTC links
+    window.navigator.registerProtocolHandler('bitcoin', '/#/open/%s', 'Blockchain')
+
+    // Register BCH links
+    window.navigator.registerProtocolHandler('web+bitcoincash', '/#/open/%s', 'Blockchain')
```

Comment:
just curious: should we think about generic solution here for other coins or not really?

Reply:
I don't think so. I've just added it to the sagas because that's something we can refactor in the future.